tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -59,7 +59,7 @@
     });
 
     function is_test_context() {
-      return window.__wptrunner_message_queue !== undefined;
+      return !!window.__wptrunner_is_test_context;
```

Comment:
If `__wptrunner_is_test_context` is not accurate, we should stick with the old one?

And in `message-queue.js` now it checks both `__wptrunner_message_queue` and `__wptrunner_process_next_event`, maybe we should be consistent between the two.